vm/devices/get/guest_emulation_device/src/lib.rs

```diff
@@ -790,6 +802,55 @@ impl<T: RingMem + Unpin> GedChannel<T> {
         Ok(())
     }
 
+    fn handle_igvm_attest(
+        &mut self,
+        message_buf: &[u8],
+        state: &mut GuestEmulationDevice,
+    ) -> Result<(), Error> {
+        let request =
+            IgvmAttestRequest::read_from_prefix(message_buf).ok_or(Error::MessageTooSmall)?;
+
+        // Request sanitization (match GED behavior)
+        if request.agent_data_length as usize > request.agent_data.len()
+            || request.report_length as usize > request.report.len()
+            || request.number_gpa as usize > get_protocol::IGVM_ATTEST_MSG_MAX_SHARED_GPA
+        {
+            Err(Error::InvalidIgvmAttestRequest)?
+        }
+
+        let request_payload = IgvmAttestRequestHeader::read_from_prefix(&request.report)
+            .ok_or(Error::MessageTooSmall)?;
+
+        let response = match request_payload.request_type {
+            IgvmAttestRequestType::AK_CERT_REQUEST => {
+                let data = vec![0xab; 2500];
```

Comment:
please document that this current implementation is a stub, used only for test scenarios